src/esbmc/options.cpp

```diff
@@ -349,6 +349,9 @@ const struct group_opt_templ all_cmd_options[] = {
      "exclude still reachable objects from --memory-leak-check"},
     {"nan-check", NULL, "check floating-point for NaN"},
     {"memory-leak-check", NULL, "enable memory leak check"},
+    {"no-memory-cleanup-check",
```

Comment:
Could you please also include it in the `no-standard-check?`? 

No-standard-check disables all basic internal checks in the ESBMC